### ISSUE TITLE
Testing metapackage for system and subsystem level black-box testing

### ISSUE THREAD

**Issue by SteveMacenski** [ros-planning/navigation2#154]
Meta package containing packages like nav2_planning_tests for each subsystem, as well as system level integration tests. 

This metapackage I would recommend branching into a separate repository

**Comment by mjeronimo** [comment]
@SteveMacenski While I don't think we're ready yet to start splitting out packages into separate repos, I do think we need a plan for this that we can discuss and implement (at a good time).

**Comment by SteveMacenski** [comment]
Agreed. I think this is a ticket for a discussion and when more testing resources exist, we revisit on what's most appropriate
